LICENSE

```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright {yyyy} {name of copyright owner}
+   Copyright 2024 Super iPaaS Integration LLC, an IBM Company
```

Comment:
The `LICENSE` file shouldn't change.

Reply:
You sure?  I assume the template portion of the license file was always intended to be filled in and just wasn't

Reply:
fixed